src/devices/OpenHardwareMonitor/SensorType.cs

```diff
@@ -0,0 +1,20 @@
+﻿#pragma warning disable CS1591
```

Comment:
header

Reply:
And why deactivation the /// comments?  I don't see any reason why

Reply:
I customary add this pragma during development, so that I don't need to write the documentation before I even know whether the API works the way I start it. But of course, this should never be in a PR ;-)